src/UserProfileService/Utilities/UserContextStore.cs

```diff
@@ -29,57 +28,18 @@ public UserContextStore(
     {
         _logger = logger;
         _context = contextAccessor.HttpContext;
-        _readService = readService;
         _userId = new Lazy<Task<string>>(GetUserIdInternal);
     }
 
     private async Task<string> GetUserIdInternal()
```

Comment:
Task as a return type is not necessary any more.

Reply:
I removed the Task as return type.